src/Common/src/CoreLib/System/SpanHelpers.T.cs

```diff
@@ -3,11 +3,14 @@
 // See the LICENSE file in the project root for more information.
```

Comment:
Should this be in its own `.char.cs` file? We are likely going to have quite a few char-specific SpanHelpers.

Reply:
I think it should, as soon as we add char-specific helpers for the other APIs (for example the IndexOf variants). We can move them when we get a 2nd helper method.